### PR TITLE
Remove home icon from default navbar

### PR DESCRIPTION
**Pull request by hadley**
@jayhesselberth @maelle does this make sense to you?

### CODE REVIEW

**Comment by hadley** [comment]
Also need to update docs, esp `build_site()`.

File: NEWS.md [hunk 1]
```diff
@@ -25,6 +29,7 @@
 
 * `\special{}` support inside `\usage{}` added to allow non-standard R usage
   syntax (@klmr, #1345).
+>>>>>>> cc9c7b1be8573f8af3453f6e5d0422d86c916701
```
Comment (maelle) [inline NEWS.md:32]: you missed this

**Review by maelle — COMMENTED** [review]
Yes it makes sense!

File: NEWS.md (hunk 1 above)
Comment (hadley) [reply NEWS.md:32]: Oops, thanks!

**Pull request merged**